Tutorials/Python_Developer_Tutorials/Readme.md

```diff
@@ -0,0 +1,34 @@
+
+# Faraday Python Developer Tutorials
+
+This tutorial series is focused on introducing Faraday as a digital radio platform to program using Python. The reader will begin by learning fundamental transmissions of commands to both local and remote (using RF) devices. The tutorial series will conclude by building upon each skill and tool-set learned to develop a basic radio data  transmission program capable of transmitting data, text, etc... of arbitrary size from one Faraday to another through wireless transmissions.
+
+**Key Concepts**
+
+* Digital Communications Fundamentals
+  * Frame/packet, error detection, fragmentation, encapsulation, ARQ (automatic retry-request)
+* Programming with Faraday tool-sets/modules
+* Building more advanced Faraday host computer programs
+
+> These tutorials rely on the currently implemented commands/functionality of Faraday and thus provide a great platform for experimentation. Higher data throughput and functionality can be achieved using other tool-sets or in conjunction with more specialized firmware programming that will be covered at a later date.
+
+##Introduction To Faraday Proxy Programming
```

Comment:
## Blocking
* I would label what is complete and what is not with a little note i.e

**[Commanding - Local](foundation/Commanding-Local/)** (Outdated)